src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -356,6 +357,22 @@ public function write($messages, $newline = false, $type = self::OUTPUT_NORMAL)
     }
 
     /**
+     * @return QuestionHelper|null
```

Comment:
Maybe just create a SymfonyQuestionHelper in `__construct()`, and keep `getQuestionHelp()`

for example:
```
    /**
     * @param InputInterface  $input
     * @param OutputInterface $output
     */
    public function __construct(InputInterface $input, OutputInterface $output)
    {
        $this->input = $input;
        $this->bufferedOutput = new BufferedOutput($output->getVerbosity(), false, clone $output->getFormatter());
        // Windows cmd wraps lines as soon as the terminal width is reached, whether there are following chars or not.
        $this->lineLength = min($this->getTerminalWidth() - (int) (DIRECTORY_SEPARATOR === '\\'), self::MAX_LINE_LENGTH);

+       $this->questionHelper = new SymfonyQuestionHelper();

        parent::__construct($output);
    }
```